docs/kane-ai-using-parameters.md

```diff
@@ -41,4 +41,88 @@ import {YOUR_LAMBDATEST_USERNAME, YOUR_LAMBDATEST_ACCESS_KEY} from "@site/src/co
         }]
       })
     }}
-></script>
\ No newline at end of file
+></script>
+This guide provides a detailed walkthrough for utilizing parameters in KaneAI to streamline test case management and execute tests with various data inputs. By leveraging parameters such as `name`, `age`, and `phone_number`, testers and developers can manage configurations effortlessly. Follow the step-by-step instructions below for successful implementation.
+
+## Key Features of Parameters in KaneAI
+- **Dynamic Data Input :** Use parameters to define and input test data dynamically during test execution.
+- **Multiple Configurations :** Simplify the execution of tests with different data sets.
+- **Seamless Test Management :** Centralized management of parameters improves efficiency and reduces redundancy.
+
+## Step 1: Define Parameters in a Web Agent Test
+- Navigate to the specific web agent test URL.
+- Prepare the test to accept inputs for `name`, `age`, and `phone_number` as parameters.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/parameters/1.png').default} alt="Image" className="doc_img"/>
+
+## Step 2: Create the Phone Number Parameter
+- Confirm that parameters for `name` and `age` are already defined in the system. To add a new parameter for `phone_number`, use one of the following methods to define the parameter:
+  - Type a **`slash (/)`** and select the Parameter option.
+  - Click the Plus icon and choose the Parameter icon.
+
+Assign a name such as `phone_number` and enter a default value (e.g., 123-456-7890). Click **Create Parameter** to save.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/parameters/3.png').default} alt="Image" className="doc_img"/>
+
+## Step 3: Finalize the Parameter
+Reference any parameter using **$curly braces syntax** (e.g.,`${phone_number}`). Enter the name in its designated input field.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/parameters/4.png').default} alt="Image" className="doc_img"/>
+
+## Step 4: Input Additional Data
+Input values for the age and phone number fields.
+Execute the test as usual.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/parameters/5.png').default} alt="Image" className="doc_img"/>
+
+## Step 5: Submit and Review Parameters
+- Access the parameter list for navigation.
+- Click Submit to confirm the data inputs.
+- Finalize by clicking Finish Test.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/parameters/6.png').default} alt="Image" className="doc_img"/>
+
+## Step 6: Test Summary Page
```

Comment:
Remove all steps and make them sections @amanchopra1905

Reply:
@Private1647 If we do that, it might cause problems in understanding; please review the new update and then suggest further